lib/util/patterns/letters.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * @fileoverview Pattern for detecting any letter (even letters outside of
+ * ASCII).
+ * @author Kevin Partington
+ */
```

Comment:
Since this file was generated using a tool, I'd just throw a sentence in here saying that this file was generated by whatever tool generated it and not to edit by hand.